pvlib/modelchain.py

```diff
@@ -1626,17 +1632,23 @@ def run_model_from_poa(self, data):
             ``'module_temperature'`` is provided, `temperature_model` must be
             ``'sapm'``.
 
-            If `data` is a tuple it must have the same length as the number
-            of Arrays in the system being modeled. Each element should specify
-            the POA (and other input) to the corresponding Array in the system
-            being modeled.  For example, ``data[0]`` contains POA irradiance
-            for ``system.arrays[0]``. Each element must satisfy the
-            requirements described above.
+            If the ModelChain's PVSystem has multiple arrays, `data` must be a
+            list or tuple with the same length and order as the PVsystem's
```

Comment:
I'm not sure "and order" is clear. I don't really have a better option than describing the correspondence as I did previously though.

Reply:
"ordering"? All I mean is that the elements in `data` must be in the same order as the arrays in `PVSystem.arrays` since we're not matching the elements to arrays with a key.

Reply:
Maybe it is fine combined with a good description/example in `modelchain.rst`

Reply:
`modelchain.rst` needs a rewrite not only to describe using a `ModelChain` with multiple `Array`s but also for the collection of results into `ModelChain.results`.  My next task. In a separate PR to your branch than this one.